addons/project/views/project_portal_project_task_templates.xml

```diff
@@ -34,7 +34,7 @@
                 <t t-foreach="grouped_tasks" t-as="tasks">
                     <tbody t-if="tasks">
                         <tr t-if="not groupby == 'none'" class="table-light">
-                            <th t-if="groupby == 'project' and multiple_projects" t-attf-colspan="{{ grouped_tasks_colspan }}">
+                            <th t-if="groupby == 'project' and multiple_projects" t-attf-colspan="{{grouped_tasks_colspan - (3 if tasks.project_id and 'allow_timesheets' in tasks.project_id and tasks.project_id.allow_timesheets else 0)}}">
```

Comment:
Isn't it possible to leave the colspan as is and override it in `hr_timesheet`?

Reply:
done :)